.circleci/config.yml

```diff
@@ -4,7 +4,8 @@ version: 2.1
 stdenv: &stdenv
   environment:
     GOCACHE: &gocache /tmp/go-build
-    IMAGE: &image saschagrunert/criocircle:1.1.0
+    IMAGE: &image quay.io/haircommander/cri-o-go-1.12:latest
+    IMAGELEGACY: &imagelegacy quay.io/haircommander/cri-o-go-1.10:latest
```

Comment:
Can we versionize the images to `1.2.0`? Or do we want to pick another location where we all have write permissions?

Reply:
added them to quay.io/crio, which I think @mrunalp gave you write access :)

Reply:
Hm, I'm not sure If I have write access to this organization, my quay.io  account exists just since a couple of days: https://quay.io/user/saschagrunert :man_shrugging:

Reply:
I pinged @mrunalp you should be added soon :)